src/utils/headers.js

```diff
@@ -86,10 +86,10 @@ const parseHeadersFile = function (filePath) {
 
   let path
   let rules = {}
-  for (const { line, index } of lines) {
+  lines.forEach(({ line, index }) => {
```

Comment:
I find the previous code easier to understand (`continue` vs `return`).

Reply:
Yeah. `return` in `forEach` is a bit confusing.